01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http')
+
+const server = http.createServer((req,res) =>{
+    if(req.url === '/') {
+        res.end('Welcome to our home Pag')
```

Comment:
```suggestion
        res.end('Welcome to our home page')
```
small typo here, no biggie